io.github.Fontforge.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: io.github.Fontforge
+
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+
+rename-icon: fontforge
+rename-desktop-file: fontforge.desktop
+
+command: fontforge
+
+finish-args:
+  - --socket=x11
+  # The file picker does not use portals :(
+  - --filesystem=host
+
+modules:
```

Comment:
Add `cleanup` (global or per-module) to remove files like headers or static libraries.